app/presenters/return-requirements/add-note.presenter.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+/**
+ * Formats data for the `/return-requirements/{sessionId}/add-note` page
+ * @module AddNotePresenter
+ */
+
+function go (session) {
+  const { id, data } = session
+
+  const pageData = {
+    id,
+    licenceRef: session.data.licence.licenceRef,
+    note: data.note.content || ''
```

Comment:
🍬 😮

Reply:
This would normally work, but it was causing nunjucks to have a wobbly, so I reverted to ternary :(